### PR TITLE
Typography

### PR DESCRIPTION
**Pull request by talrejah02**
please review and give suggestion

### CODE REVIEW

File: components/common-css/container.css [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+code{
+margin: 1rem;
+margin-top: 0;
+margin-bottom: 0;
+background-color: var(--primary);
+font-size: 0.8rem;
+letter-spacing: 1px;
+line-height: 1.2rem;
+border-radius: 7px;
+padding-left: 1rem !important;
```
Comment (ShreyashMahajan) [inline components/common-css/container.css:63]: try not to use important

File: components/common-css/drawer.css [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+}
+.list-item{
+    display: flex;
+    align-items: center;
+    font-weight: 400;
+    font-size: 0.8rem;
+    color: var(--primary);
+    margin: 0.5rem;
+    margin-left:0;
+    margin-right: 0;
```
Comment (ShreyashMahajan) [inline components/common-css/drawer.css:49]: you can use shorthand property of margin here

**Review by ShreyashMahajan — APPROVED** [review]
Good to merge

**Review by someshtiwarii — APPROVED** [review]
good to go !

**Pull request merged**